src/site/filters/liquid.js

```diff
@@ -209,6 +209,7 @@ module.exports = function registerFilters() {
   };
 
   liquid.filters.formatSharableLinkID = (id, description) => {
+    if (!id) return '';
     if (!description) return id;
```

Comment:
Smart how you return `id` here instead! 🙂 Would you mind writing a unit test or 2 for `formatShareableLinkID` if they don't exist already when you have time?

Reply:
done!